tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -395,73 +474,60 @@ def monitor_rebuild(self, mon_map_cmd):
-                cmd=f"ceph-authtool /tmp/keyring --import-keyring /tmp/{k_file[-1]}",
-                pod_obj=mon_obj,
+        self._exec_oc_cmd(
+            cmd=(
+                f"cp -n {constants.OPENSHIFT_STORAGE_NAMESPACE} "
+                f"{mon_a.name}:/var/lib/ceph/mon/ceph-a/store.db "
+                f"{self.backup_dir}/"
             )
+        )
+        logger.info("Copying store.db to rest of the monitors")
+        for mon in get_mon_pods(namespace=config.ENV_DATA["cluster_namespace"]):
+            if not mon.get().get("metadata").get("labels").get("ceph_daemon_id") == "a":
```

Ruff (0.15.15)

[warning] Use `mon.get().get("metadata").get("labels").get("ceph_daemon_id") != "a"` instead of `not mon.get().get("metadata").get("labels").get("ceph_daemon_id") == "a"`

Replace with `!=` operator

(SIM201)